Makefile

```diff
@@ -351,4 +351,16 @@ test:
 	skipper make $(SKIPPER_PARAMS) _test
 
 _test: $(REPORTS)
+	if [ -d /tmp/assisted_test_infra_logs ]; then rm -rf /tmp/assisted_test_infra_logs; fi
+	mkdir /tmp/assisted_test_infra_logs
+	cp -p discovery-infra/test_infra/tools/tf_network_pool.json /tmp/tf_network_pool.json
 	python3 -m pytest $(or ${TEST},discovery-infra/tests) -k $(or ${TEST_FUNC},'') -m $(or ${TEST_MARKER},'') --verbose -s --junit-xml=$(REPORTS)/unittest.xml
+
+test_parallel:
+	if [ -d /tmp/assisted_test_infra_logs ]; then rm -rf /tmp/assisted_test_infra_logs; fi
+	mkdir /tmp/assisted_test_infra_logs
+	cp -p discovery-infra/test_infra/tools/tf_network_pool.json /tmp/tf_network_pool.json
+	skipper make $(SKIPPER_PARAMS) _test_parallel
+
+_test_parallel: $(REPORTS)
+	python3 -m pytest -n $(or ${MULTI_NUM}, '2') $(or ${TEST},discovery-infra/tests) -k $(or ${TEST_FUNC},'') -m $(or ${TEST_MARKER},'') --verbose -s --junit-xml=$(REPORTS)/unittest.xml
```

Comment:
The reason is to be able to run also a single test during development for example.
With -n option, no live logs until test are finished. 
Also, -n parameter cannot be ' ' so its either set, or not exist at all

Reply:
so unable to merge, and ill change the name to TEST_WORKERS_NUM

Reply:
but isn't `-n` can't be equal to 1?
What happens when trying to run a single test in parallel? Does it crush?

Reply:
no it wont crush it will work, just that you wont have live logs until test is PASSED\FAILED. 
So need to be able to run tests without -n

Reply:
Can you check this workaround?
https://stackoverflow.com/questions/27006884/pytest-xdist-without-capturing-output

Reply:
didn't help so solve :(  same output

Reply:
Well not exactly the same, its now show ERROR on real time but doesn't help much. 
I noticed on the web that when using -n that you also get progress bar - but when i run multi tests - no progress bar. 
I also tried to use a plugin called sugar, but it looks like all "graphical" output are somehow filtered, maybe because we run it from within a container.